load/UpdateDBPTextFilesets.py

```diff
@@ -61,32 +61,45 @@ def validateFileset(self, subTypeCode, bibleId, filesetId, languageRecord, lptsI
 			fullFilesetPath,
 			self.config.directory_accepted,
 			databaseName, iso3, iso1, direction]
+		print("================= Invoke BiblePublisher ========================")
+		print("cmd: ", cmd)
+		print("=========================================")
 		response = subprocess.run(cmd, stderr=subprocess.PIPE, stdout=subprocess.PIPE, timeout=120)
 		if response == None or response.returncode != 0:
 			return((Log.EROR, "BiblePublisher: " + str(response.stderr.decode("utf-8"))))
 		print("BiblePublisher:", str(response.stdout.decode("utf-8")))	
```

Comment:
change string to start with "BiblePublisher invocation successful"

Reply:
@bradflood , it has been updated.